lib/ops/opshandler/opshandler.go

```diff
@@ -114,15 +112,20 @@ func NewWebHandler(cfg WebHandlerConfig) (*WebHandler, error) {
 	h.GET("/portal/v1/currentuserinfo", h.needsAuth(h.getCurrentUserInfo))
 	h.POST("/portal/v1/users", h.needsAuth(h.createUser))
 	h.DELETE("/portal/v1/users/:user_email", h.needsAuth(h.deleteLocalUser))
+	h.PUT("/portal/v1/accounts/:account_id/sites/:site_domain/users/:user_email", h.needsAuth(h.updateUser))
 
 	// API keys API
 	h.POST("/portal/v1/apikeys/user/:user_email", h.needsAuth(h.createAPIKey))
 	h.GET("/portal/v1/apikeys/user/:user_email", h.needsAuth(h.getAPIKeys))
 	h.DELETE("/portal/v1/apikeys/user/:user_email/:api_key", h.needsAuth(h.deleteAPIKey))
 
-	// Tokens
+	// Invites API
+	h.POST("/portal/v1/accounts/:account_id/sites/:site_domain/tokens/userinvites", h.needsAuth(h.createUserInvite))
```

Comment:
can we get rid of account_id in these modified APIs? I have been passing dummy values for some time now.

Reply:
No, we can't. Keep in mind this is ops handler, not web handler. A pair of accountID/clusterID is used to route request to appropriate cluster. In web handlers there's no account ID IIRC.

Reply:
You are right, for these endpoints we do not require accountID on webapi level but we do for some others which is out of scope of this PR.